### PR TITLE
[zh] Sync Getting started with the Kubernetes Gateway API blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync Getting started with the Kubernetes Gateway API blog into Chinese:
```
content\zh\blog\2022\getting-started-gtwapi\index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
`您` 可以适当少一些，读起来有点啰嗦 -_!

**Comment by wilsonwu** [comment]
> `您` 可以适当少一些，读起来有点啰嗦 -_!

No problem, I will improve it soon.

File: content/zh/blog/2022/getting-started-gtwapi/index.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
+---
+title: Kubernetes Gateway API 入门
+description: 使用 Gateway API 为您的 Kubernetes 集群配置入口流量。
+publishdate: 2022-12-14
+attribution: Frank Budinsky (IBM)
+keywords: [traffic-management,gateway,gateway-api,api,gamma,sig-network]
+---
+
+无论您使用 Istio 或其他服务网格运行您的 Kubernetes 应用程序服务，
+还是只是在 Kubernetes 集群中使用普通服务，
```
[the remark this answers is not in the document]
Comment (wilsonwu) [reply content/zh/blog/2022/getting-started-gtwapi/index.md]: Fixed in the latest commit.

File: content/zh/blog/2022/getting-started-gtwapi/index.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+您都需要为集群外部的客户端提供对您应用程序服务的访问方式。
+如果您使用的是普通 Kubernetes 集群，您可能正在使用 Kubernetes
+[Ingress](https://kubernetes.io/zh-cn/docs/concepts/services-networking/ingress/) 资源来配置入口流量。
+
+一段时间以来，人们都知道 Kubernetes Ingress 资源存在重大缺陷，
+尤其是在使用它为大型应用程序配置入口流量以及使用 HTTP 以外协议时问题更为突出。
```
[the remark this answers is not in the document]
Comment (wilsonwu) [reply content/zh/blog/2022/getting-started-gtwapi/index.md]: Fixed in the latest commit.

File: content/zh/blog/2022/getting-started-gtwapi/index.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+其中一个问题是它在单个资源中同时配置了客户端 L4-L6 属性（例如端口、TLS 等）和服务端
+L7 路由，而对于大型应用程序的配置应该由不同的团队在不同的命名空间中进行管理。
+此外，通过尝试在不同 HTTP 代理之间描绘一个共同点，Ingress 只能支持最基本的
```
[the remark this answers is not in the document]
Comment (wilsonwu) [reply content/zh/blog/2022/getting-started-gtwapi/index.md]: Fixed in the latest commit.

**Review by windsonsea — APPROVED** [review]
Thanks.

**Pull request merged**